### DISCUSSION TITLE
Got rejected even though my proof contain the headmaster's sign

### DISCUSSION THREAD

**Discussion by ShikiHTM** [community/community#53093] [GitHub Education]
Hi, I'm a student of THPT Phan Ngọc Hiển from Vietnam
I want to get GitHub student benefits, Even though I uploaded my good student selection exam paper from my **school** which contains my name, school name, birthday, class, and headmaster's sign. I still get rejected

Note: I've uploaded my transcript previous and yeh still rejected
Moderators Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by samus-aran** [comment]
HI @ShikiHTM 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍
